ui/src/app/features/face-services/face-verification/verification-result/verification-result.component.ts

```diff
@@ -0,0 +1,154 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { Component, ElementRef, Input, OnDestroy, ViewChild, OnChanges, SimpleChanges, Output, EventEmitter } from '@angular/core';
+import { Observable, Subscription } from 'rxjs';
+import { map, tap } from 'rxjs/operators';
+import {
+  getImageSize,
+  ImageSize,
+  recalculateFaceCoordinate,
+  resultRecognitionFormatter,
+  createDefaultImage,
+} from '../../face-services.helpers';
+import { RequestResult } from '../../../../data/interfaces/response-result';
+import { RequestInfo } from '../../../../data/interfaces/request-info';
+import { VerificationServiceFields } from '../../../../data/enums/model.enum';
+
+@Component({
+  selector: 'app-verification-result',
+  templateUrl: './verification-result.component.html',
+  styleUrls: ['./verification-result.component.scss'],
+})
+export class VerificationResultComponent implements OnChanges, OnDestroy {
+  @Input() processFile: File;
+  @Input() checkFile: File;
+  @Input() requestInfo: RequestInfo;
+  @Input() printData: RequestResult;
+  @Input() files: Observable<any>;
+  @Input() isLoaded: boolean;
+  @Input() pending: boolean;
+  @Output() selectProcessFile = new EventEmitter();
+  @Output() selectCheckFile = new EventEmitter();
+
+  @ViewChild('processFileCanvasElement') set processFileCanvasElement(canvas: ElementRef) {
+    if (canvas) {
+      if (this.printSubscription1) {
+        this.printSubscription1.unsubscribe();
+      }
+      if (this.processFile) {
+        this.printSubscription1 = this.printResult(
+          canvas,
+          this.processFileCanvasSize,
+          this.processFile,
+          this.printData,
+          VerificationServiceFields.processFileData
+        ).subscribe();
+      }
+    }
+  }
+
+  @ViewChild('checkFileCanvasElement') set checkFileCanvasElement(canvas: ElementRef) {
+    if (canvas) {
+      if (this.printSubscription2) {
+        this.printSubscription2.unsubscribe();
+      }
+      if (this.checkFile) {
+        this.printSubscription2 = this.printResult(
+          canvas,
+          this.checkFileCanvasSize,
+          this.checkFile,
+          this.printData,
+          VerificationServiceFields.checkFileData
+        ).subscribe();
+      }
+    }
+  }
+
+  processFileCanvasSize: ImageSize = { width: 500, height: null };
+  checkFileCanvasSize: ImageSize = { width: 500, height: null };
+  faceDescriptionHeight = 25;
+  formattedResult: string;
+
+  private printSubscription1: Subscription;
```

Comment:
instead of definining 2 subs, only 1 can be used
```
private subcsriptions: Subscription;
subscription.add(<second subscription...>)
subscription.unsubscribe()
```